### ISSUE TITLE
Actions and ability to store (as a variable) an item or track name for use within cycle actions

### ISSUE THREAD

**Issue by Jeff0S** [reaper-oss/sws#625]
_From [musicbynumbers@gmail.com](https://code.google.com/u/musicbynumbers@gmail.com/) on February 06, 2014 18:54:49_

This might be tricky to do but if we could an action like this;

-SWS: take selected item/track name and store as variable for use in cycle actions (note: only works with single item selections?)

And then in cycle actions, when creating an action that includes either reaconsole, label processor (or hopefully "find window" at some point). We could then add a symbol (or something) to represent and include this copied name/variable within that action. 

Basically, a way to insert it into the current action so we can create custom cycle actions that pull this name from the item and then use it to do things like select all other items that have the same name in the project and lots of other cool things I'm too tired to think of right now! ;)

Hope that makes sense and I know it might lead to some circular arguments etc but could be very useful! :)

_Original issue: http://code.google.com/p/sws-extension/issues/detail?id=625_
